index.html

```diff
@@ -439,6 +447,69 @@ <h2>Legend</h2>
        }
      }
 
+     // Sorting
+     function setupSorting() {
+       const table = document.querySelector("table");
+
+       const headers = table.querySelectorAll("th");
+
+       headers.forEach((header, columnIndex) => {
+         if (!header.classList.contains('sortable')) {
+           return;
+         }
+         const button = document.createElement("button");
+         const isInitDescending = header.classList.contains("more-info");
+         const sortNumerically = isInitDescending;
+         button.setAttribute("aria-label", `Sort by ${header.textContent}`);
+         button.textContent = header.textContent;
+         header.setAttribute("aria-label", header.textContent);
+         header.textContent = '';
+         header.append(button);
+         if (isInitDescending) {
+           header.setAttribute("aria-sort", "descending");
+         }
```

Comment:
From the code, it wasn't clear to me that this column was:
a) reverse-sorted to begin with
b) effectively the primary sort column for the table

A comment to that effect might help.